server/fileupdater.mjs

```diff
@@ -488,4 +489,13 @@ function v16UpdateCountParameter(routine) {
       }
     }
   }
-}
\ No newline at end of file
+}
+
+function v17MaterialSymbols(properties) {
+  if (properties.type === 'classes') {
```

Comment:
`properties` is basically the widget JSON. So `properties.classes` and `properties.id` etc.